bindings/ros/README.md

```diff
@@ -0,0 +1,40 @@
+# ROS Wrapper for the ADI ToF library
+
+## Overview
+This ROS package facilitates depth and IR data acquisition and processing for the Analog Devices depth cameras.
+
+## Installation
+
+- **Install the recommended [ROS distribution](http://wiki.ros.org/Distributions) for your operating system**
+  - [ROS Install page](http://wiki.ros.org/ROS/Installation)
+- **[Optional] Install additional ROS packages needed to build and run the examples**
```

Comment:
I'm not sure if the "[Optional]" tag is a good choice, please let me know if you have a better idea.
Its purpose is to let the users know that they need to check if these packages are installed , or install them otherwise .I also chose the word optional because in this case, the rviz package is included in the Debian package ros-melodic-desktop-full needed to install ros, but it is not included in the ros base(Bare Bones) package ros-melodic-ros-base.

Reply:
Why not just say: "Install rviz if not already installed"?

Reply:
Because the list of packages may get longer.I think I would better change it to "Make sure you have these packages installed before you run the examples".